### ISSUE TITLE
Pet battle tab ignores prat settings

### ISSUE THREAD

**Issue by Zasz333** [Legacy-of-Sylvanaar/prat-3-0#100]
This has bothering me for years because of the way Ive set up my chat.

I use two chat windows. One for general chat, guild, group and whispers and the other window for loot and system messages. All visible, no tab buttons visible, everything goes invisible after about 15 seconds.

But when I enter a pet battle the game creates a pet battle tab and that tab is always visible as a tab on my main chat window. It doesnt care that I disable the tab button in the settings which would make it invisible for me so I wouldnt care that it exists. But since Im not doing pet battles often anymore I just close the tab after pet battles.

I wrote a ticket in pandaria and got a GM response with an admission that the tab is not behaving like it should. Probably because it pops up again and again and cant be prevented to pop up. I think I asked this years ago on curse too but I dont remember if I got a response. Is there something that can be done now?

**Comment by plusmouse** [comment]
Are you using the ChatTabs module?

That should hide the tabs as wanted - I've just tested without the module and the tab remains visible but a bit faded, with the module on it fades entirely.

**Comment by Zasz333** [comment]
Yes, Im using it to hide all other chat tabs. But when that particular tab appears it ignores the settings that all other tabs follow.

**Comment by plusmouse** [comment]
I'm going to need detailed steps to replicate the bug, as I can't cause the glitchy behaviour to happen

Is the intended behaviour for the tab to _close_ rather than have the visible tab fade to invisible? I get the battle pet tab staying open until I `/reload`.

**Comment by Zasz333** [comment]
The pet battle tab is normally not there until the first pet battle. With prat and my settings the tab buttons are not supposed to be visible. Once I start a pet battle that tab is automatically added to the default chat window after the default tabs like general chat and combatlog. The tab is always visible and it doesnt show up in prat next to the other tabs that I have & can hide or make visible. The tab shows up and just sits there.

I expect it to show up in prat so I can hide the tab like all the others.

I dont know how else to explain it.
